_posts/2019-06-24-logging-call-into-your-service-with-ringcentral-embeddable.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title:  "Logging call into your service with RingCentral Embeddable"
+date:   2019-06-24 10:00:00 +0800
+categories: blog
+author: Embbnux Ji
+---
+
+![hubspot call logging with Embeddable](https://github.com/ringcentral/hubspot-embeddable-ringcentral-phone/raw/master/docs/img/screenshots/hs6.png)
+
+In previous articles, we show how to use RingCentral Embeddable to integrate RingCentral services into your website. After integrating web phone feature, you must want to log call into your service. This article will show you how to log call information with RingCentral Embeddable's third party service API.
+
+Related Links:
+
+* RingCentral Embeddable: [Github Repository](https://github.com/ringcentral/ringcentral-embeddable)
+* Third Party service API: [Document](https://github.com/ringcentral/ringcentral-embeddable/blob/master/docs/third-party-service-in-widget.md#log-call-into-your-service)
+
+
+## Integrate RingCentral Embeddable
+
+We provides `Adapter JS way` to help developers to integrate RingCentral Embeddable. Just need to add following script into your website's header:
+
+```html
+<script>
+  (function() {
+    var rcs = document.createElement("script");
+    rcs.src = "https://ringcentral.github.io/ringcentral-embeddable/adapter.js";
+    var rcs0 = document.getElementsByTagName("script")[0];
+    rcs0.parentNode.insertBefore(rcs, rcs0);
+  })();
+</script>
+```
+
+The script will create a RingCentral widget into your website, so your customer can login with RingCentral account to access RingCentral services:
```

Comment:
`This` script will `inject` a RingCentral widget into your website, so your `customers` can login with ....